ObjC/Encoder+ObjC.mm

```diff
@@ -25,7 +25,7 @@
 
 
 @interface NSObject (FleeceInternal)
-- (void) fl_encodeTo: (Encoder*)enc;
+- (void) fl_encodeToFLEncoder: (FLEncoder)enc;
```

Comment:
This was already declared in Fleece+CoreFoundation.h. You can just remove this`@interface`.